1.0.0-ANNOUNCEMENT.md

```diff
@@ -0,0 +1,43 @@
+# Announcing TUF 1.0.0
+
+In the past year we have made an effort to revise, redesign and rewrite this
+python-tuf reference implementation, and we are very excited to announce a
+stable 1.0.0 release scheduled for January 2022. The release *will* include:
+- a modern low-level [*metadata
+  API*](https://theupdateframework.readthedocs.io/en/latest/api/tuf.api.html)
+- a fully specification-compliant [*updater
+  client*](https://theupdateframework.readthedocs.io/en/latest/api/tuf.ngclient.html),
+  serving as a more robust and yet more flexible stand-in replacement
+  for the legacy client updater
+
+As discussed in [ADR 2](docs/adr/0002-pre-1-0-deprecation-strategy.md), this
+release *will not* include any legacy code, as its maintenance has become
+infeasible for the python-tuf team. The pre-1.0.0 deprecation strategy from ADR
+2 applies as follows:
+
+> *Bugs reported with tuf versions prior to 1.0.0 will likely not be addressed
+directly by tuf’s maintainers. Pull Requests to fix bugs in the last release
+prior to 1.0.0 will be considered, and merged (subject to normal review
+processes). Note that there may be delays due to the lack of developer resources
+for reviewing such pull requests.*
+
+For the reasons outlined in [ADR 10](docs/adr/0010-repository-library-design.md
+), this release *will not* include a new *repository tool*. However, the new
```

Comment:
In response to requests for mentioning the planned work as per ADR 10, what about:

```suggestion
), this release *will not yet* include a new *repository tool*. However, the new
```
?

Reply:
@MVrachev @jku